test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/PackageItemListViewModelTests.cs

```diff
@@ -37,5 +58,401 @@ public void LocalSources_PackageReader_NotNull()
             PackageReaderBase reader = func();
             Assert.IsType(typeof(PackageArchiveReader), reader);
         }
+
+        [Fact]
+        public void IconUrl_WithMalformedUrlScheme_ReturnsDefault()
+        {
+            var iconUrl = new Uri("httphttphttp://fake.com/image.png");
```

Comment:
Use `fake.test` instead of `fake.com`.  Always prefer the [.test domain](https://en.wikipedia.org/wiki/.test) in tests unless otherwise necessary.

Check the remainder of this file too.

Reply:
my friend who is a bigwig in the domain selling business, worked for 10 years to enable new TLD (top level domains) like .test - he'd give you a big hug for advocating this. :-)

Reply:
although it likely changes the behavior... testing two different things.
fake.com exists...but image.png isn't found on it, while fake.test is a unknown domain.

fixed in next iteration.